package.json

```
@@ -2,7 +2,7 @@
   "author": "Transcend Inc.",
   "name": "@transcend-io/privacy-types",
   "description": "Core enums and types that can be useful when interacting with Transcend's public APIs.",
-  "version": "1.13.1",
+  "version": "1.13.2",
```

```suggestion
  "version": "1.14.0",
```

would do a minor version bump for this